arithmetization/src/test/java/net/consensys/linea/zktracer/ReplayTests.java

```diff
@@ -199,84 +203,346 @@ void replayMultipleReplayFiles() {
     replayBulk("");
   }
 
+  // @Disabled
   @Test
   void leoFailingRange() {
-    replay("5389571-5389577.json.gz");
+    replay("5389571-5389577.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void failingMmuModexp() {
-    replay("5995162.json.gz");
+    replay("5995162.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void failRlpAddress() {
-    replay("5995097.json.gz");
+    replay("5995097.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void rlprcptManyTopicsWoLogData() {
     replay("6569423.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void multipleFailingCallToEcrecover() {
-    replay("5000544.json.gz");
+    replay("5000544.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void incident777zkGethMainnet() {
-    replay("7461019-7461030.json.gz");
+    replay("7461019-7461030.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void issue1006() {
-    replay("6032696-6032699.json.gz");
+    replay("6032696-6032699.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void issue1004() {
     replay("6020023-6020029.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void block_6110045() {
     // The purpose of this test is to check the mechanism for spotting divergence between the replay
     // tests and mainnet.  Specifically, this replay has transaction result information embedded
     // within it.
-    replay("6110045.json.gz");
+    replay("6110045.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void failingCreate2() {
-    replay("2250197-2250197.json.gz");
+    replay("2250197-2250197.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void blockHash1() {
-    replay("8718090.json.gz");
+    replay("8718090.json.gz", false);
   }
 
+  // @Disabled
   @Test
   void blockHash2() {
-    replay("8718330.json.gz");
+    replay("8718330.json.gz", false);
   }
 
   // TODO: should be replaced by a unit test triggering AnyToRamWithPadding (mixed case) MMU
   // instruction
+  // @Disabled
   @Test
   void negativeNumberOfMmioInstruction() {
-    replay("6029454-6029459.json.gz");
+    replay("6029454-6029459.json.gz", false);
   }
 
   @Test
   void simpleSelfDestruct() {
-    replay("50020-50029.json.gz");
+    replay("50020-50029.json.gz", false);
   }
 
   // TODO: should be replaced by a unit test triggering a failed CREATE2
   @Test
   void failedCreate2() {
-    replay("41640-41649.json.gz");
+    replay("41640-41649.json.gz", false);
+  }
+
+  // Leo's range split up 5104800-5104883
```

Comment:
could be nice to have a descriptive name of the test to know what was the issue, and let us have the possibility to run unit test to reduce the nb of replay tests. CI will be too long if not ...

Reply:
Let me see what I can do.